docs/development/test-vectors.rst

```diff
@@ -625,6 +625,108 @@ Custom PKCS12 Test Vectors
 * ``pkcs12/cert-aes256cbc-no-key.p12`` - A PKCS12 file containing a cert
   (``x509/custom/ca/ca.pem``) encrypted via AES 256 CBC with the
   password ``cryptography`` and no private key.
+* ``pkcs12/no-name-no-pwd.p12`` - A PKCS12 file containing a cert
```

Comment:
I can review these vectors and land them as a separate PR if you are willing to split them out and then rebase this PR against main once it merges.

Reply:
@reaperhulk that would be great, thank you!